chef/automate
# Client Runs Suggestion terms have upper case

### :nut_and_bolt: Description: What code changed, and why?
Updating config-mgmt-service suggestions to return case sensitive terms. This reintroduced the elasticsearch field extension ".lower". When this extension is used it matches values case-insensitively. This needs to be used on all node-state index filter requests. This can not be used on converge-history index requests or node-state index sort parameters. 

This change required a reindex of the node-state index. 

### :chains: Related Resources
#1396 

### :+1: Definition of Done
When using the Client Runs search bar upper case terms are displayed with upper case. 

### :athletic_shoe: How to Build and Test the Change
1. `build components/config-mgmt-service && build components/ingest-service && start_all_services`
1. Add some nodes with `chef_load_nodes 100`
1. Open the Client Runs page https://a2-dev.test/infrastructure/client-runs
1. In the search bar, select the "Chef Organization" type and then type one letter. 
1. Ensure there are organizations with Upper case letters. 
1. To test uppercase cookbooks run `send_chef_run_example`
1. In the Client Runs search bar, select the "Cookbook" type and then type "ap".
1. Ensure one of the suggestions has uppercase "APT". 

### :white_check_mark: Checklist


### :camera: Screenshots, if applicable
